wsd/DocumentBroker.cpp

```diff
@@ -1849,7 +1852,7 @@ bool DocumentBroker::autoSave(const bool force, const bool dontSaveIfUnmodified)
     // Note: a loaded view cannot be disconnecting.
     const auto itLastEditingSession = _sessions.find(_lastEditingSessionId);
     const std::string savingSessionId =
-        (itLastEditingSession != _sessions.end() && itLastEditingSession->second->isReadOnly() &&
+        (itLastEditingSession != _sessions.end() && itLastEditingSession->second->isWritable() &&
```

Comment:
from isReadOnly -> isWritable - looks like a logic error - do we miss a not ?

Reply:
It was a bug, evidently, since a read-only session shouldn't be used for saving (and indeed it couldn't have been the last-editing-session in the first place, so `isReadOnly()` would be returning false anyway, and we'd fall back to choose a writable session).